Classes/room.py

```diff
@@ -199,27 +202,129 @@ def isValidExit(self, exit):
                 return True
         return False
 
-    def addItem(self, item):
+    def addDroppedItem(self, item):
+        """
+        Adds a dropped item to the room.
+        This should be used when the player "drops" items.
+        Input "item" should be Item object.
+        """
+        self.droppedItems.append(item)
+
+    def removeDroppedItem(self, itemName):
+        """
+        Removes a dropped item from the room.
+        This should be used when the player "picks up" a dropped item.
+        Input "itemName" should be an item's name string.
+        Returns the item if removed, or 1 if item wasn't found.
+        """
+        for item in self.droppedItems:
+            if item.name == itemName:
+                self.droppedItems.remove(item)
+                return item
+        return 1

```

Comment:
The point of the Return 1 is for error handling. Return None can work, but I usually use error codes since they give an explicit result, and because sometimes there can be more than one or two possible results to expect (even though that's not the case here).

Reply:
In Python, I think the general convention is to use try/except blocks for that sort of error handling. Exception handling can be used to get any explicit details for errors when they occur.

Reply:
Yes, but you also can't necessarily always rely on exceptions being raised either. Either one works here though, so I can change it.

Reply:
Let me know what you decide. If you'd rather just keep it as-is I can approve and merge.

Reply:
It's been changed.

Reply:
Thanks!